### PR TITLE
EVG-1935 allow terminating hosts from cli

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: cli/host.go [hunk 1]
```diff
@@ -103,10 +107,42 @@ func (cmd *HostStatusCommand) Execute(_ []string) error {
...
+	}
+
+	data := struct {
+		HostID string `json:"host_id"`
+		Action string `json:"action"`
+	}{cmd.HostID, "terminate"}
+
+	rPipe, wPipe := io.Pipe()
+	encoder := json.NewEncoder(wPipe)
+	go func() {
```
Comment (bsamek) [inline cli/host.go]: Why is this in a separate goroutine?

File: cli/http.go [hunk 2]
```diff
@@ -108,8 +111,10 @@ func (ac *APIClient) doReq(method, path string, apiVersion int, body io.Reader)
 		req, err = http.NewRequest(method, fmt.Sprintf("%s/%s", ac.APIRoot, path), body)
 	} else if apiVersion == 2 {
 		req, err = http.NewRequest(method, fmt.Sprintf("%s/%s", ac.APIRootV2, path), body)
+	} else if apiVersion == -1 {
```
Comment (bsamek) [inline cli/http.go]: Can this be made available in the API?

File: cli/host.go (hunk 1 above)
Comment (john-m-liu) [reply cli/host.go]: This was copied from other CLI code that formed request bodies. I tried it in the main goroutine but that caused the encode method to block indefinitely

File: cli/http.go (hunk 2 above)
Comment (john-m-liu) [reply cli/http.go]: This route is only used by the UI server at the moment and requires the evergreen settings

**Pull request merged**